src/services/ai-server/ai-persona-service/ai.persona.service.entity.ts

```diff
@@ -24,8 +25,8 @@ export class AiPersonaService
   @Column('varchar', { length: ENUM_LENGTH, nullable: false })
   dataAccessMode!: AiPersonaDataAccessMode;
 
-  @Column('text', { nullable: false })
-  prompt!: string;
+  @Column('simple-array', { nullable: false })
+  prompt!: string[];
```

Comment:
pretty sure that's not generated in the migration

Reply:
Typeorm stores simple arrays as text fields with comma separated values, so no DB change needed.

Reply:
Changed this to `simple-json` in order to handle prompt messages with `,`; Added data migration statement to the migration.